src/ts/ucp.ts

```diff
@@ -29,7 +29,8 @@ import 'tinymce/themes/silver';
 import 'tinymce/themes/mobile';
 
 $(document).ready(function() {
-  $.ajaxSetup({
+    let type = 'experiments_templates';
```

Comment:
is this really needed? Why not just add it directly on line 83. It doesn't look like it's used elsewhere.

Reply:
Has been removed and the constant inserted in line 83.